hudi-client/hudi-spark-client/src/test/java/org/apache/hudi/table/functional/TestMarkerBasedRollbackStrategy.java

```diff
@@ -118,7 +118,7 @@ public void testMarkerBasedRollbackAppend() throws Exception {
     String f0 = testTable.addRequestedCommit("000")
         .getFileIdsWithBaseFilesInPartitions("partA").get("partA");
     testTable.forCommit("001")
-        .withLogMarkerFile("partA", f0, IOType.CREATE);
+        .withLogMarkerFile("000", "partA", f0, IOType.APPEND, 1);
```

Comment:
Is the test running on table version 8 with APPEND log marker?

Reply:
`MarkerBasedRollbackStrategy` considers all types of markers in general; nevertheless created HUDI-9244 for test clean-up as a follow-up.